### DISCUSSION TITLE
"install Lingmo OS" not disappear after installation.

### DISCUSSION THREAD

**Discussion by vampywiz17** [LingmoOS/Community#14] [Bugs Report/缺陷提交] [answered]
After install, Installer still here:

![image](https://github.com/user-attachments/assets/7278aea1-913a-42fa-a4d0-13a52c3475c4)

**Comment by elysia-best** [comment]
Got it. @lingmo-dream

**Reply by vampywiz17** [reply, marked as answer]
new update are fixed it :) Thanks!
